src/alert/lsst.rs

```diff
@@ -466,35 +466,51 @@ pub struct DiaForcedSource {
 pub struct ForcedPhot {
     #[serde(flatten)]
     pub dia_forced_source: DiaForcedSource,
-    pub magpsf: f32,
-    pub sigmapsf: f32,
+    pub magpsf: Option<f32>,
+    pub sigmapsf: Option<f32>,
     pub diffmaglim: f32,
-    pub isdiffpos: bool,
-    pub snr: f32,
+    pub isdiffpos: Option<bool>,
+    pub snr: Option<f32>,
 }
 
 impl TryFrom<DiaForcedSource> for ForcedPhot {
     type Error = AlertError;
     fn try_from(dia_forced_source: DiaForcedSource) -> Result<Self, Self::Error> {
-        let psf_flux = dia_forced_source
-            .psf_flux
-            .ok_or(AlertError::MissingFluxPSF)?
-            * 1e-9;
         let psf_flux_err = dia_forced_source
             .psf_flux_err
             .ok_or(AlertError::MissingFluxPSF)?
             * 1e-9;
 
-        let (magpsf, sigmapsf) = flux2mag(psf_flux.abs(), psf_flux_err);
-        let diffmaglim = fluxerr2diffmaglim(psf_flux_err);
+        // for now, we only consider positive detections (flux positive) as detections
+        // may revisit this later
+        let (magpsf, sigmapsf, isdiffpos, snr) = match dia_forced_source.psf_flux {
+            Some(psf_flux) => {
+                let psf_flux = psf_flux * 1e-9;
+                if (psf_flux / psf_flux_err) > 3.0 {
```

Comment:
This is false for `psf_flux < 0`, which I think is what you want, but I just want to make sure.

Also, I wonder if the 3.0 needs to be a `const`, just to help describe what this value is and to help prevent mistakes. This applies to ZTF, too. And I assume we want both surveys to use the same SNR threshold, so the `const` should be accessible to both modules.

Reply:
yes that's what I wanted, for now let's only consider positive incursions as detections which is what we care about for transients. At a later date we might revisit this for stellar folks, but yeah just making an absolute value would make a negative incursion look like its brightening which is a problem.

Reply:
and I like the idea of the 3 being a const! then we can have it somewhere at the top of the file (or in another enum-dedicated file) with a comment. Could call it SNT, for (Signal to Noise Threshold)

Reply:
done